backend/src/main/java/com/huellapositiva/infrastructure/orm/repository/JpaStatusRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.huellapositiva.infrastructure.orm.repository;
+
+import com.huellapositiva.infrastructure.orm.entities.JpaStatus;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public interface JpaStatusRepository extends JpaRepository<JpaStatus, Integer> {
```

Comment:
StatusRepository is very ambiguous. What status are we referring to? I would call it JpaProposalStatus otherwise anyone can think it is the status of a volunteer, esal or any other kind of thing

Reply:
Yes, we agree. Changed applied.